docs/usage.md

```diff
@@ -1,153 +1,306 @@
+#### 5. Variant calling - Structural variants
 
-Note that multiple profiles can be loaded, for example: `-profile test,docker` - the order of arguments is important!
-They are loaded in sequence, so later profiles can overwrite earlier profiles.
+| Mandatory | Optional   |
+| --------- | ---------- |
+|           | target_bed |
+|           | bwa_index  |
 
-If `-profile` is not specified, the pipeline will run locally and expect all software to be installed and available on the `PATH`. This is _not_ recommended, since it can lead to different results on different machines dependent on the computer enviroment.
+#### 6. SNV annotation & Ranking
 
-- `test`
-  - A profile with a complete configuration for automated testing
-  - Includes links to test data so needs no other parameters
-- `docker`
-  - A generic configuration profile to be used with [Docker](https://docker.com/)
-- `singularity`
-  - A generic configuration profile to be used with [Singularity](https://sylabs.io/docs/)
-- `podman`
-  - A generic configuration profile to be used with [Podman](https://podman.io/)
-- `shifter`
-  - A generic configuration profile to be used with [Shifter](https://nersc.gitlab.io/development/shifter/how-to-use/)
-- `charliecloud`
-  - A generic configuration profile to be used with [Charliecloud](https://hpc.github.io/charliecloud/)
-- `conda`
-  - A generic configuration profile to be used with [Conda](https://conda.io/docs/). Please only use Conda as a last resort i.e. when it's not possible to run the pipeline with Docker, Singularity, Podman, Shifter or Charliecloud.
+| Mandatory                     | Optional                       |
+| ----------------------------- | ------------------------------ |
+| genome<sup>1</sup>            | gnomad_af<sup>4</sup>          |
+| vcfanno_resources<sup>2</sup> | reduced_penetrance<sup>5</sup> |
+| vcfanno_toml<sup>3</sup>      | vcfanno_lua                    |
+| vep_cache_version             | vep_filters                    |
+| vep_cache                     |                                |
 
-### `-resume`
+<sup>1</sup>Genome version is used by vep. You have the option to choose between GRCh37 and GRCh38.
+<sup>2</sup>Path to vcf files and their indices used by vcfanno. Sample file [here](https://github.com/ramprasadn/test-datasets/blob/raredisease/reference/vcfanno_resources.txt).
+<sup>3</sup>Path to a vcfanno configration file. Sample file [here](https://github.com/ramprasadn/test-datasets/blob/raredisease/reference/vcfanno_config.toml).
+<sup>4</sup>Gnomad vcf file can be downloaded from here https://gnomad.broadinstitute.org/downloads.
+<sup>5</sup>Used by genmod while ranking the variants. Sample file [here](https://github.com/ramprasadn/test-datasets/blob/raredisease/reference/reduced_penetrance.tsv).
```

Comment:
Line breaks. Also some of these paths points to your fork of the test-dataset repo